src/Checkout.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\ShopStandards\Checkout.
+ */
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Checkout related settings and actions.
+ */
+class Checkout {
```

Comment:
The proposed structure looks good, but I'd strongly recommend to scope the classes into features and thus have a class for Salutation only.

If anything generic remains, we can keep a generic Checkout class for that.